database/migrations/2018_06_04_129200_create_app_timeout_action_executed_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class CreateAppTimeoutActionExecutedTable extends Migration
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('APP_TIMEOUT_ACTION_EXECUTED', function (Blueprint $table) {
```

Comment:
Update these variable to use the new style. PRO_UID is process_id and USR_UID is user_id, etc.